lib/modules/platform/codecommit/index.ts

```diff
@@ -273,7 +274,7 @@ export async function getPr(
     title: prInfo.title!,
     targetBranch: prInfo.pullRequestTargets![0].destinationReference!,
     destinationCommit: prInfo.pullRequestTargets![0].destinationCommit!,
-    sha: prInfo.revisionId,
+    sha: prInfo.revisionId as LongCommitSha, // Not sure about this one ... maybe not a problem, but something to check
```

Comment:
TODO: let's check this before merge

Reply:
@PhilipAbed is `revisionId` definitely a long/full SHA? I know we already call it `sha` but I have doubts if it definitely is.

API docs are.. inconclusive

<img width="480" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/e085cde4-317a-4e35-97e6-9d749b7367e8">

<img width="674" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/5f1ace90-05ef-44d4-ad81-375d6890bea3">

Reply:
Resolved: removed